### ISSUE TITLE
Handle missing family column in germline study cache update

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#1921]
If the family column is not present in a germline study, attempting to update the cache results in a crash with `list indices must be integers or slices, not NoneType`.

I'm not 100% sure if we require for the column to be there or not. Technically, I would be able to replace the family ID automatically with the source ID is not present. I'd appreciate some feedback from experts of germline studies and our ISA-Tab template experts on what is the best way to proceed.

**Comment by mikkonie** [comment]
As a personal note, the crash happens in the germline study plugin `_update_study_cache()`, specifically here:

```
fam_idx = get_index_by_header(assay_table, 'family')
# ..
row_fam = row[fam_idx]['value']  # fam_idx ends up None if column is absent, hilarity ensues
```

**Comment by mikkonie** [comment]
It has been confirmed to me that the column is expected to be mandatory. I should still make handling it more graceful in the plugin: if index is not found, log missing column in timeline event.

**Comment by mikkonie** [comment]
I looked into this more closely and elsewhere we do also support not having a family column. This seems like a straighforward bug. Good thing is it's also straightforward to fix, so I'll get that in for v0.14.2.

**Comment by mikkonie** [comment]
Fixed.
